domains/potatopioneers.is-a.dev

```diff
@@ -0,0 +1,18 @@
+{
+    "description": "potatoPioneer's website",
+    "repo": "https://github.com/potatoPioneers/potatopioneers.site",
+    "owner": {
+        "username": "ponfertato",
+        "email": "ponfertato@ya.ru"
+        "github": "ponfertato"
+        "gitlab": "ponfertato"
+        "telegram": "ponfertato"
```

Comment:
Remove this line

Reply:
Removed information other than email address